x-pack/plugins/endpoint/server/index_pattern.ts

```diff
@@ -74,4 +75,8 @@ export class IngestIndexPatternRetriever implements IndexPatternRetriever {
       throw new Error(errMsg);
     }
   }
+
+  getPolicyResponseIndexPattern(ctx: RequestHandlerContext): Promise<string> {
+    return Promise.resolve('metrics-endpoint-policy-*');
```

Comment:
See comment above, but I think this needs to be `metrics-endpoint.policy-default-1`

Also this is temporary until we get it into the package right?

Reply:
yes, once in the package we will update it. We will tackle this as part of this ticket https://github.com/elastic/endpoint-app-team/issues/395

Reply:
Sounds good 👍